css/src/components/form/radio.scss

```diff
@@ -0,0 +1,117 @@
+$radio-circle-size: 1.25em !default;
+$radio-dot-size: 0.625em !default;
+$radio-dot-checked-color: $primary !default;
+$radio-dot-color: $text-subtle !default;
+$radio-timing-function: $input-timing-function !default;
+$radio-duration: $input-transition-duration !default;
+$radio-animation: boop !default;
+$radios-spacing: 0.5em !default;
+
+@mixin radio-dot {
+	position: absolute;
+	inset: 0;
+	width: $radio-dot-size;
+	height: $radio-dot-size;
+	margin: auto;
+	border-radius: 50%;
```

Comment:
nit: rounded radius variable courtesy of @ollips's recent PR?

Reply:
Updated